genic/save.c

```diff
@@ -125,13 +125,13 @@ void saveheader(BigFile * bf) {
             (big_block_set_attr(&bheader, "Time", &InitTime, "f8", 1)) ||
             (big_block_set_attr(&bheader, "Redshift", &redshift, "f8", 1)) ||
             (big_block_set_attr(&bheader, "BoxSize", &Box, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaM", &Omega, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaB", &OmegaBaryon, "f8", 1)) ||
-            (big_block_set_attr(&bheader, "OmegaL", &OmegaLambda, "f8", 1)) ||
+            (big_block_set_attr(&bheader, "OmegaM", &CP.Omega0, "f8", 1)) ||
```

Comment:
Ye-es...but neutrinos I think interpolate between UR and CDM (or is it NCDM?) for CLASS, depending on their temperature, which isn't what we want here. In S-GenIC I have Omega, OmegaLambda and OmegaBaryon, which I like. Then I specify the total neutrino mass and the hierarchy and compute OmegaNeutrino internally.